app/src/main/java/com/example/fooding/activities/SignupActivity.java

```diff
@@ -67,22 +68,22 @@ public void done(ParseException e) {
                     if (e == null) {
                         Log.i(TAG, "onClick signUp button");
                         Toast.makeText(getApplicationContext(),"Signed up sucessfully" , Toast.LENGTH_SHORT);
-
+                        goLoginActivity();
                     }
 
                     else {
                         Log.i(TAG, "onClick Error");
                         Toast.makeText(getApplicationContext(),"Error Signing Up" , Toast.LENGTH_SHORT);
                         Log.e(TAG, e.toString());
                     }
-                    goLoginActivity();
+//                    goLoginActivity();
```

Comment:
don't commit comments unless they have a TODO

Reply:
I forgot to delete the comment, I won't commit that next time.